t/t6025-merge-symlinks.sh

```diff
@@ -40,7 +39,7 @@ test_expect_success 'merge master into b-symlink, which has a different symbolic
 '
 
 test_expect_success 'the merge result must be a file' '
-    test -f symlink
+    test_path_is_file symlink
```

Comment:
It probably makes sense to [split this commit](https://git-scm.com/docs/git-rebase#_splitting_commits) into two: the first one avoids the pipes, the second one replaces the `test -f` invocations (those are two separate concerns, therefore they should be addressed by two separate commits, too).

Oh, and please do not forget to fix the `Signed-off-by` line: there should be a `-` instead of a space before the `by:`.